Gruntfile.js

```diff
@@ -26,6 +27,103 @@ module.exports = function (grunt) {
             'Licensed GPLv2 */\n',
         // Task configuration.
 
+        config: {
+            local: {
+                options: {
+                    variables: {
+                        'METADATA_HOST': 'localhost:3000',
```

Comment:
EVA_HOST and EVA_VERSION better than METADATA, as it also contains variant data.